api_docs/README.rst

```diff
@@ -0,0 +1,25 @@
+The api-docs are seriealised from source
+using sphinx for code gen and ".rst" for source formatting.
+
+@pedromorgan admits that I hate both, but we have to live with ;-)
+
+Snags...
+- sphinx does not automatically load a whole path.. so the `fab
+- Why not use readthedocs- because we cant load pyqt etc
+
+So this set is generated locally,
+and then uploaded.. idea is to NOT have ghpages,
+but instead a decicated app, fast release, docs autgen etc..
```

Comment:
Don't blame you for wanting to have faster releases and all that fancy stuff.

My life's been currently cluttered with all sorts of things, but I don't have a unified web presence to announce such to the world. 

(( ^ that and I'm a bit of a hermit announcing IRL hiccups to everyone in the world ))

Other than that note, I support the notion of faster development. I just need to watch the PRs closely to give better feedback on them.